function testFunc() {
  try {
    console.log('hi');
  } catch (error) {}
}

console.log(testFunc());
